activities/zio-flow-twilio/shared/src/main/scala/zio/flow/activities/twilio/Message.scala

```diff
@@ -0,0 +1,54 @@
+package zio.flow.activities.twilio
+
+import zio.flow._
+import zio.schema.DeriveSchema
+
+final case class Message(
+  body: String,
+  num_segments: String,
+  direction: String, // TODO: should be Direction serialized into a single string field
+  from: PhoneNumber,
+  to: PhoneNumber,
+  date_updated: String, // TODO: should be Instant and parsed using DateTimeFormatter.RFC_1123_DATE_TIME
+  price: Option[String],
+  error_message: Option[String],
+  uri: String,
+  account_sid: AccountSid,
+  num_media: String,
+  status: String, // TODO: should be Status serialized into a single string field
+  messaging_service_sid: MessagingServiceSid,
+  sid: MessageSid,
+  date_sent: String,    // TODO: should be Instant and parsed using DateTimeFormatter.RFC_1123_DATE_TIME
+  date_created: String, // TODO: should be Instant and parsed using DateTimeFormatter.RFC_1123_DATE_TIME
+  error_code: Option[Int],
+  price_unit: Option[String],
+  api_version: String
+  //subresource_uris:  Map[String, String] // TODO: this should be a dynamic Json object
```

Comment:
This is also something that should have this type so the workflow can do something with it (once there is remote syntax for `Map[K, V]`) but adding it here generates the wrong json codec (it expects an array not an object).

Reply:
This would be `DynamicValue`, right, or should be? That's the closet thing to JSON.

We should make sure ZIO Schema has special support in JSON codec for String-valued maps.